.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
Is there a reason we want to cover all Node.js versions? It'll result in slower builds.
If we're not doing anything low level Node.js we should probably limit to the last 2 LTS.

Reply:
12 is still officially supported by node.js so makes sense to leave it in. i typically just match whatever core supports

Reply:
removed https://github.com/metaplex/arweave-cost/pull/2